client/src/components/FavCheckbox.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from '@emotion/styled';
+// import CheckmarkPath from '../icons/CheckmarkIcon';
```

Comment:
Outcommented code is unnecessary code

Reply:
I removed it for now, maybe I'll experiment with some changes of the Checkbox later. thank you.